src/lib/math/pcurves/pcurves_impl/pcurves_impl.h

```diff
@@ -698,14 +705,48 @@ class ProjectiveCurvePoint {
       }
 
       constexpr Self dbl_n(size_t n) const {
-         // TODO it is possible to optimize this by carrying over values from
-         // the previous iteration into the next
+         if constexpr(Self::A_is_minus_3) {
+            /*
+            Repeated doubling for A == -3
+            Cost: 2S + *2 + n*(4S + 4M + 2*2 + 1*3 + 4A)
```

Comment:
```
2S + *2 + n*(4S + 4M + 2*2 + 1*3 + 4A)
      ^ typo?
```

Reply:
Implicit one *2 but yeah everywhere else that is explicit

Reply:
Fixed